src/usdb_syncer/gui/mw.py

```diff
@@ -58,6 +59,7 @@ def __init__(self) -> None:
         self._setup_search()
         self._setup_download()
         self._setup_signals()
+        self.logger = get_logger(__file__)
```

Comment:
This doesn't depend on the instance, but on the module, so please make it a global like in other modules, e.g. sync_meta.py.

Reply:
I am so sorry. 3 weeks ago I was not able to find any use of a `get_logger`-method in the project so I thought you mean the `__file__` part. I don't know why I could not find it, because on the 2nd glance yesterday I easily found it. But then I was not sure if you really mean `global` like outside the class, because I saw we used `self.logger = get_logger(__file__)` as instance attribute somewhere else. Hence I decided to do so too 🙈 

My fault, I will change it to what you actually already suggested in the very first comment.

Reply:
No problem. Often loggers belong to a specific song id. Then it makes sense to define them on an instance that also represents a single song, like `Context` in song_loader.py.

Reply:
done